src/status_im2/navigation/view.cljs

```diff
@@ -49,25 +52,39 @@
    (when top?
      {:padding-top (safe-area/get-top)})))
 
+(defn screen-wrapper
+  [{:keys [insets background-color component sheet? screen-theme]}]
+  (let [user-theme (quo2.theme/get-theme)]
+    [theme/provider (or screen-theme user-theme)
+     [rn/view {:style (wrapped-screen-style insets background-color)}
+      [inactive]
+      (if sheet?
+        [:f> bottom-sheet-screen/f-view component]
```

Comment:
Not for this PR, Just a reminder, We need this solution for the `bottom-sheet` too. For instance, the `How to scan` bottom sheet is a dark theme by default, and the `Request to join` community sheet is user themed.

Reply:
Yep absolutely, tried to keep this pr focused on the bare minimum to get it working but you are right in that we will need to add it there too!